src/MockFactory.ts

```diff
@@ -64,111 +65,146 @@ const executeOperation = (
 
 interface Options {
   mocks?: ResolverMap
+  addTypename?: Boolean
+}
+
+interface MockOptions<TVariables> {
+  variables?: TVariables
+  mocks?: ResolverMap
+  addTypename?: Boolean
 }
 
 class MockFactory {
   private schema: GraphQLSchema
   private mocks: ResolverMap
+  private addTypename: Boolean
 
   private mockSchema(options: { mocks?: ResolverMap } = {}) {
     const clonedSchema = transformSchema(this.schema, [])
-    const mergedMocks = mergeResolvers(this.mocks, options.mocks || {})
+    const mergedMocks = mergeResolvers(this.mocks, options.mocks ?? {})
```

Comment:
Why is the `??` operator better here?

Reply:
I can't honestly remember why I changed this. I must have done it for a reason but I can't think of a reason the `||` wouldn't work here just as good

Reply:
I changed it back and my memory was jogged. Eslint was complaining with the new rules I added: [prefer-nullish-coalescing](https://github.com/typescript-eslint/typescript-eslint/blob/v4.12.0/packages/eslint-plugin/docs/rules/prefer-nullish-coalescing.md)